content/doc/book/managing/index.adoc

```diff
@@ -131,7 +133,14 @@ link:/doc/book/managing/script-console/[Script Console]::
 Execute an Apache Groovy script for administration, troubleshooting, and diagnostics.
 
 Prepare for Shutdown::
-Prevents new builds from starting so that the system can be shut down safely.
+Prevents new builds from starting so that the system can be shut down safely. Displays a red headband with a custom
+message so that users know what is about to happen.
+
+image:managing/prepare-for-shutdown.png[Red headband with a custom message]
+
+WARNING: This does not ask Jenkins to stop; this action will just prevent new builds from starting. If you need to stop
+or restart Jenkins, you should use the command line or the `/restart` and `/safeRestart` end points. There is also a
+plugin called https://plugins.jenkins.io/saferestart/[Safe Restart] that will add a `Restart Safely` link in the UI.
```

Comment:
The general pattern has been to use sentence per line for new documentation additions so that it is easier to review changes in context of a single sentence.  Optional, not a blocking change.

I'm more accustomed to the word "banner" to describe the red text block, rather than "headband".  Also not a blocking comment.

```suggestion
Prevents new builds from starting so that the system can be shut down safely.
Displays a red banner with a custom message so that users know what is about to happen.

image:managing/prepare-for-shutdown.png[Red headband with a custom message]

[NOTE]
This does not ask Jenkins to stop; this action will just prevent new builds from starting.
If you need to stop or restart Jenkins, you should use the command line or the `/restart` and `/safeRestart` end points.
There is also a plugin called https://plugins.jenkins.io/saferestart/[Safe Restart] that will add a `Restart Safely` link in the UI.
```

When the change is ready for review by others, you'll want to take it out of "draft" and remove the "[WIP]" from the pull request title.